index.js

```diff
@@ -14,9 +14,10 @@ const isFromWorker = (payload) => Boolean(payload.fromWorker);
 
 const createFingerprint = () => crypto.randomBytes(64).toString('hex');
 
-const createWorkerContent = (jobCode) => `
-  (${jobCode})();
-`;
+const createWorkerContent = (jobCode) => {
+  const hoverlordFile = __filename.replace(/\\/g, '\\\\'); // in case of windows system
```

Comment:
Maybe we could declare `/\\/g` and `'\\\\'` as UNIX_REGEX and WINDOWS_REGEX, I'm not thinking about other names at the moment. What do you think about that?

Reply:
Just another proposal:

```js
const createWorkerContent = (jobCode) => {
  return `(${jobCode})(require(${JSON.stringify(__filename)}));`;
}
```

In this way all backslashes are safely escaped if there. 
This is a more generic solution due the fact that we have to pass the code representation of a string: that's exactly what JSON.stringify does.

Reply:
Better off this way, thanks!